manimo/actuators/arms/franka_arm.py

```diff
@@ -8,7 +8,7 @@
 from manimo.actuators.arms.robot_ik.robot_ik_solver import RobotIKSolver
 
 # from manimo.actuators.arms.moma_arm import MujocoArmModel
-from manimo.actuators.controllers.policies import CartesianPDPolicy, JointPDPolicy
+from manimo.actuators.controllers.policies import CartesianPDPolicy, JointPDPolicy, OperationalSpaceLowFreq
```

Comment:
remove osc, not used in neural mp anyways

Reply:
fixed